### PR TITLE
Run monitor docker containers under the configured user.

### PR DESCRIPTION
**Pull request by aleksbykov**

### CODE REVIEW

**Comment by bentsi** [comment]
@aleksbykov per our discussion you should run the `start-all.sh` not under user root
(run `bash -cxe` without sudo) and additionaly please remove improve the script in order not to copy paste the whole script that only difference between OS-s is how we start the docker

File: sdcm/remote.py [hunk 1]
```diff
@@ -169,19 +170,33 @@ def __init__(self, hostname, user="root", port=22, connect_timeout=60, password=
...
+        self.connect_config = {'key_filename': os.path.expanduser(self.key_file)}
+        self._create_connection(self.hostname,
                                 user=self.user,
                                 port=self.port,
-                                config=config,
+                                config=self.ssh_config,
+                                connect_timeout=self.connect_timeout,
+                                connect_kwargs=self.connect_config)
+
+    @retrying()
```
Comment (bentsi) [inline sdcm/remote.py]: add parameters explicitly

Comment (aleksbykov) [reply sdcm/remote.py]: fixed

File: sdcm/cluster.py [hunk 2]
```diff
@@ -3109,20 +3114,22 @@ def start_scylla_monitoring(self, node):
         if node.is_rhel_like() or node.is_ubuntu16():
             run_script = dedent("""
                 cd {0.monitor_install_path}
-                sudo systemctl start docker
                 mkdir -p {0.monitoring_data_dir}
-                chmod 777 {0.monitoring_data_dir}
-                ./start-all.sh -s {0.monitoring_conf_dir}/scylla_servers.yml -n {0.monitoring_conf_dir}/node_exporter_servers.yml -d {0.monitoring_data_dir} -l -v {0.monitoring_version} -b "-web.enable-admin-api"
+                ./start-all.sh \
+                -s {0.monitoring_conf_dir}/scylla_servers.yml \
+                -n {0.monitoring_conf_dir}/node_exporter_servers.yml \
```
Comment (bentsi) [inline sdcm/cluster.py]: it looks like it the same command for all OS-s

Comment (aleksbykov) [reply sdcm/cluster.py]: yes, you right. They become the same

**Pull request merged**